azure-pipelines.yml

```diff
@@ -0,0 +1,39 @@
+trigger:
+  branches:
+    include:
+      - ni/current
+
+pool:
+  name: DevCentral-Docker
```

Comment:
For @mure and @kjohn1922's benefit: the machines in this pool run Linux, so they don't use docker desktop. We found that docker on linux doesn't run into the memory issues that we saw on windows, so we're unblocked.